doc/release-notes/6807-binder.md

```diff
@@ -0,0 +1 @@
+Binder has been added as an external tool. From the dataset page, you can launch Binder, which spins up a computational environment in which you can explore the code and data in the dataset, or write new code, such as a Jupyter notebook.
```

Comment:
small tweak - can this say "as an available external tool" or something similar? To make it obvious that it a) has to be configured, b) is optional

Reply:
@scolapasta ah, good point. Fixed in 4a68a23ab6.